deploy/7_AddBassetInBasketManager.ts

```diff
@@ -0,0 +1,44 @@
+import { constants } from "ethers";
+import { DeployFunction } from "hardhat-deploy/types";
+
+const func: DeployFunction = async ({
+  deployments,
+  getNamedAccounts,
+}) => {
+  const network = deployments.getNetworkName();
+  const {deployer} = await getNamedAccounts();
+  let bAssets = [
+    "0x6b41566353d6c7b8c2a7931d498f11489dacac29", // ZUSD Testnet
+    "0xcb46c0ddc60d18efeb0e586c17af6ea36452dae0", // DOC Tesnet
+  ];
+  let factors = [1,1];
+  let bridges = [constants.AddressZero, constants.AddressZero];
+  let mins = [10,10];
+  let maxs = [100,100];
```

Comment:
maxs and mins should be null. pls verify if it works correctly. there should not be any restrictions except 1:1 exchange.

Reply:
updated in this [commit](https://github.com/DistributedCollective/mynt/pull/5/commits/b791ecb597f6586e57a2ee71466f6b0984cb8889)